Source/Protocol.c

```diff
@@ -22,10 +22,37 @@
 #include <string.h>
 #include <time.h>
 
+#ifdef WIN32
```

Comment:
Including Windows api just for 1 function call. Like grrrrr. Let migw handle that.

Reply:
`#define WIN32_LEAN_AND_MEAN` and we're good :)